### ISSUE TITLE
Generating a resource should also generate a DeleteOperation

### ISSUE THREAD

**Issue by jwoertink** [luckyframework/lucky#1499]
When you run the `lucky gen.resource.browser User` task, it'll generate a `SaveUser` operation for you. We should also generate a `DeleteUser` operation.

https://github.com/luckyframework/lucky/blob/master/tasks/gen/resource/browser.cr

https://github.com/luckyframework/lucky/tree/master/tasks/gen/templates/resource/operations

The template should be fairly easy:

```
class Delete<%= resource %> < <%= resource %>::DeleteOperation
  # Read more on deleting records
  # https://luckyframework.org/guides/database/deleting-records
end
```

**Comment by nodeMD** [comment]
Hello 😃 
I've created a PR to address that issue: https://github.com/luckyframework/lucky/pull/1500
Please take a look and review when you will have a moment 🙂

**Comment by jwoertink** [comment]
Closed by #1500
